### PR TITLE
Update PULL_REQUEST_TEMPLATE.md

### PR DESCRIPTION
**Pull request by PippaClarkGDS**

### CODE REVIEW

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -1,7 +1,13 @@
...
 
-### Changes proposed in this pull request
+What are you trying to do? Is this something that changes how the Tech Docs Template behaves, or is it fixing a bug? 🐛
 
 
-### Guidance to review
+## Identifying a user need
+
+Do you have evidence whether the change will meet the needs of users? Let us know about any user research or testing you’ve done.
+
```
Comment (jonathanglassman) [inline .github/PULL_REQUEST_TEMPLATE.md:13]: Should we add a section on who should review this? e.g. FE dev, BE dev, someone in a specific team or with a specific set of skills acquired over a very long career?

**Review by jonathanglassman — COMMENTED** [review]
Looks good. Made one comment

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 2]
```diff
@@ -1,7 +1,13 @@
-### Context
+## Please fill in the sections below
 
+After you submit your issue, the technical writing team from Central Digital and Data Office (CDDO) will discuss and prioritise it at our fortnightly triage meeting. We’ll then let you know if and when we’ll move it forward.
```
Comment (lfdebrux) [inline .github/PULL_REQUEST_TEMPLATE.md:3]: Anything not in comments will appear in the submitted issue if not removed by the submitter, so to make things easier for them I think this part should be commented out:

```suggestion
<!--
## Please fill in the sections below

After you submit your issue, the technical writing team from Central Digital and Data Office (CDDO) will discuss and prioritise it at our fortnightly triage meeting. We’ll then let you know if and when we’ll move it forward.
-->
```

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 3]
```diff
@@ -1,7 +1,13 @@
...
+ 
+## What’s changed
 
-### Changes proposed in this pull request
+What are you trying to do? Is this something that changes how the Tech Docs Template behaves, or is it fixing a bug? 🐛
```
Comment (lfdebrux) [inline .github/PULL_REQUEST_TEMPLATE.md:7]: This should probably also be commented, as it is instructions to the user

```suggestion
<!-- What are you trying to do? Is this something that changes how the Tech Docs Template behaves, or is it fixing a bug? 🐛 -->
```

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 4]
```diff
@@ -1,7 +1,13 @@
...
 
 
-### Guidance to review
+## Identifying a user need
+
+Do you have evidence whether the change will meet the needs of users? Let us know about any user research or testing you’ve done.
```
Comment (lfdebrux) [inline .github/PULL_REQUEST_TEMPLATE.md:12]: As above

```suggestion
<!-- Do you have evidence whether the change will meet the needs of users? Let us know about any user research or testing you’ve done. -->
```

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 1 above)
Comment (PippaClarkGDS) [reply .github/PULL_REQUEST_TEMPLATE.md:15]: I'm happy to leave as is for this iteration and steer away from mandating anything just yet but happy to keep an eye on it to check the PRs are meeting a certain standard and tweak it in future if needed

**Review by lfdebrux — APPROVED** [review]

**Pull request merged**